src/microbe_stage/CompoundCloudPlane.cs

```diff
@@ -504,8 +510,8 @@ public Vector3 ConvertToWorld(int cloudX, int cloudY)
             if (compound == null)
                 break;
 
-            // Skip if compound is non-useful
-            if (!storage.IsUseful(compound))
+            // Skip if compound is non-useful or an agent
+            if (!storage.IsUseful(compound) || compound.IsAgent)
```

Comment:
I saw earlier that you removed the always useful property from the oxytoxy and didn't add it for mucilage. Would this code change be unnecessary if we marked all agents as always useful? Or another way would be to have a new property "AlwaysAbsorbed". I don't want to have this hyper special code in the compound cloud system.

Reply:
I think I do really want to keep the agents knowledge out of here. Oh and I just realized you use this to *skip* the absorption. I think we may want to have damaging agents in the future that damage cells that absorb them. So anyway a general "do not absorb" flag would be nice for the compound class so we have that flexibility in the future.